utils/common/src/main/kotlin/Extensions.kt

```diff
@@ -464,3 +464,12 @@ fun Throwable.collectMessages(): List<String> {
  * Recursively collect the messages of this [Throwable] and all its causes and join them to a single [String].
  */
 fun Throwable.collectMessagesAsString() = collectMessages().joinToString("\nCaused by: ")
+
+/**
+ * Retrieve query parameters specified by its [key].
+ */
+fun URI.getQueryParameterValue(key: String) = query.split("&").flatMap {
```

Comment:
Formatting: Please start a new line after `=`.

Reply:
Please use the single-char version of split for clarity when possible: `split('&')`